core/src/data/rasterSource.h

```diff
@@ -36,7 +36,7 @@ class RasterSource : public TileSource {
     // TODO Is this always PNG or can it also be JPEG?
     virtual const char* mimeType() override { return "image/png"; };
 
-    void loadTileData(std::shared_ptr<TileTask> _task, TileTaskCb _cb);
+    virtual void loadTileData(std::shared_ptr<TileTask> _task, TileTaskCb _cb);
 
```

Comment:
Since c++11 one can use `override` to have the compiler check that a function actually overrides a virtual function of the base class. I think this is much better than using `virtual` in the derived class just as a hint for the programmer.